src/notation/internal/notationconfiguration.cpp

```diff
@@ -151,7 +151,10 @@ void NotationConfiguration::init()
     settings()->valueChanged(USER_STYLES_PATH).onReceive(nullptr, [this](const Val& val) {
         m_userStylesPathChanged.send(val.toPath());
     });
-    fileSystem()->makePath(userStylesPath());
+
+    if (!(userStylesPath().empty())) {
```

Comment:
One code-style nitpick: for readability, I personally prefer to write conditions like this without extra parentheses; 
```suggestion
    if (!userStylesPath().empty()) {
```
(sorry 😄)

Reply:
Alright, the change is in!